backend/plugins/extensibles/j2_service_templates/interfaces_model.py

```diff
@@ -0,0 +1,32 @@
+from pydantic import BaseModel
+from typing import Dict, FrozenSet, List, Optional, Sequence, Set, Tuple, Union
+
+from backend.core.models.service import service_lifecycle
+from backend.core.models.models import queue_strat
+
+class interface_model_args(BaseModel):
+    # your model goes here!
+    hosts: list
+    username: str
+    password: str
+    driver: str
+    command: str
+
+class interface_model(BaseModel):
+    # this class MUST match the filename & the filename must be formatted $servicetemplatename_model.py
+    operation: service_lifecycle
+    args: interface_model_args
+    queue_strategy: Optional[queue_strat] = None
+
+    class Config:
+        # add an example payload under the "example" dict 
+        schema_extra = {
+            "example": {
+                "operation": "retrieve",
+                "args":{
```

Comment:
the args should also include driver & command

Reply:
Whoops forgot to commit that update. Its in now.